File: CS/MyCardView/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using MyCardView.Properties;

namespace MyCardView
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            Image image = Resources.img;
            for (int i = 0; i < 10; i++)
                dataTable1.Rows.Add("Item " + i.ToString(), image);
        }
    }
}

File: VB/MyCardView/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace MyCardView
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			Dim image As Image = My.Resources.img
			For i As Integer = 0 To 9
				dataTable1.Rows.Add("Item " & i.ToString(), image)
			Next i
		End Sub
	End Class
End Namespace
